tests/regression/36-apron/97-array_octagon.c

```diff
@@ -53,7 +95,11 @@ void example1(void) {
   // Values that may be read are 1 or 2
   assert(a[z] == 1); //UNKNOWN
   assert(a[z] == 2); //UNKNOWN
-  assert(a[z] == 0); //FAIL
+  assert(z >= 0);
+
+  // Relies on option sem.int.signed_overflow assume_none
+  assert(z <= j); //UNKNOWN
+  assert(a[z] != 0); //UNKNOWN
```

Comment:
Would this not be `FAIL` then, given that we now have ` --set sem.int.signed_overflow assume_none` ?